### DISCUSSION TITLE
Topic not appear

### DISCUSSION THREAD

**Discussion by thhtthht** [Monika-After-Story/MonikaModDev#8003] [QA - code and content creation (technical)]
I have created a submod, when I put it in game folder then open MAS, Monika don't mention. (Sorry because of my bad english)

**Comment by KritRom** [comment]
Can you send me the code please? I'll try to help as much as I can

**Reply by thhtthht** [reply]
> Can you send me the code please? I'll try to help as much as I can

Here: 

init 5 python:
    addEvent(Event(persistent.event_database,eventlabel="monika_YuGiOh",category=['games'],prompt="YuGiOh",random=True))

label monika_YuGiOh:
    m 2euc "Hey, [player]."
    m 7eua "Do you know Yu-Gi-Oh! ?"
    m 7eua "I means not anime nor manga, I'm talking about card game that based on the Duel Monsters concept from the original manga series."
    m 7wup "It's best-selling card game of all time according to Guinness World Records."
    m 7eua "In Yu-Gi-Oh!, players using a combination of monsters, spells, and traps to defeat their opponent."
    m 2eua "It's gameplay really makes me interested."
    m 2wuo "I think Yu-Gi-Oh! is more strategier than chess."
    m 2eub "If you play this game, I hope you can teach me the rule and how to play this game,"
    m 2eua "So we can have fun with each other."
    m 2eua "Ehehe~"

return

**Reply by HoshinoRuki** [reply]
I meet the same problem,do you solved the problem now ?By the way,when i define my own eventlabel,it will show error page,how you slove this problem?

**Reply by KritRom** [reply]
Sorry for late reply, can you tell me how long you waited for Moni to say the topic?

**Reply by thhtthht** [reply]
> Sorry for late reply, can you tell me how long you waited for Moni to say the topic?

she hasn't mentioned it until now
